source/03-LF/01.ptx

```diff
@@ -11,8 +11,635 @@
+        D: <m>-12</m>
+      </p>
+    </answer>
+  </task>
+   </activity>
+
+  <activity xml:id="LR-4-Investigate-slope">
+  <introduction>
+    <p>
+      Use the given graph of the function, <m>f(x)=3x-4</m>, to investigate the average rate of change of a linear function.
+    </p>
+    <image xml:id="AROC-linear-function">
+      <sageplot>
+        f(x) = 3*x-4
+        p=plot(f, (x, -10, 10), ymin=-10, ymax=10, color='blue', thickness=3, detect_poles=True)
+        p
+      </sageplot>
+    </image>
+  </introduction>
+  <task>
+    <statement>
+      <p>
+        What is the average rate of change on the interval <m>[-2,0]</m>?
+        <ol marker= "A." cols="2">
+          <li><m>\frac{1}{3}</m></li>
+          <li><m>3</m></li>
+          <li><m>-\frac{1}{7}</m></li>
+          <li><m>-7</m></li></ol>
+      </p>
+    </statement>
+    <answer>
+      <p>
+        B: <m>3</m>
+      </p>
+    </answer>
+  </task>
+  <task>
+    <statement>
+      <p>
+        What is the average rate of change on the interval <m>[-1,5]</m>?
+        <ol marker= "A." cols="2">
+          <li><m>3</m></li>
+          <li><m>\frac{1}{3}</m></li>
+          <li><m>\frac{2}{3}</m></li>
+          <li><m>-3</m></li></ol>
+      </p>
+    </statement>
+    <answer>
+      <p>
+        A: <m>3</m>
+      </p>
+    </answer>
+  </task>
+  <task>
+    <statement>
+      <p>
+        Based on your observations in parts <m>a</m> and <m>b</m>, what do you think will be the average rate of change on the interval <m>[5,25]</m>?
+      </p>
+    </statement>
+  </task>
+  </activity>
+
+  <remark>
+    Notice in <xref ref="LR-4-Investigate-slope"/>, the average rate of change was the same regardless of which interval you were given. But in <xref ref="LR-3-Calculate-AROC-from-graph"/>, the average rate of change was not the same across different intervals.
+  </remark>
+
+<definition xml:id="def-slope">
+  <statement>
+    <p>
+      The <term>slope</term> of a line is a number that represents the direction and steepness of the line. For a linear function, the slope never changes - meaning it is a constant average rate of change.
+    </p>
+  </statement>
+</definition>
+
+<activity xml:id="LR-5-Compare-Slopes-of-Lines">
+<introduction>
+  <p>
+    The steepness of a line depends on the vertical and horizontal distances between two points on the line. Use the graph below to compare the steepness, or slope, of the two lines.
+  </p>
+  <image xml:id="comparing-slopes-of-lines">
+    <sageplot>
+      f(x) = (1/2)*x-1
+      g(x) = 4*x-3
+      p=plot(f, (x, -10, 10), ymin=-10, ymax=10, color='blue', thickness=3, detect_poles=True) + point((0,-1),size=40,xmin=-10,xmax=15,ymin=-10,ymax=10)+ point((4,1),size=40,xmin=-10,xmax=15,ymin=-10,ymax=10) + plot(g, (x, -10, 10), ymin=-10, ymax=10, color='red', thickness=3, detect_poles=True) + point((0,-3),size=40,xmin=-10,xmax=15,ymin=-10,ymax=10, color='red')+ point((2,5),size=40,xmin=-10,xmax=15,ymin=-10,ymax=10, color='red')
```

Comment:
Going ahead and merging this, but may want to think about adding gridlines here.  Just add `gridlines=[[-10..10],[-10..10]]` as an extra argument of your plot command.